src/include/rados/librados.hpp

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 #include "memory.h"
 #include "buffer.h"
+#include "common/ceph_time.h"
```

Comment:
since we'll want a c version anyway, maybe it would be better to use the same type in the c++ api to avoid exposing ceph_time